core/templates/services/page-title.service.ts

```diff
@@ -28,26 +28,44 @@ export class PageTitleService {
   pageSubtitleForMobile: string = null;
   constructor(private titleService: Title) {}
 
+  /**
+   * @param {string} title - Title of the page to be set to.
```

Comment:
Add a one-line comment to explain the function as well - This function is used to...

Here and below.